[CTOR-46] [Documentation] Add Pack Teldat SNMP (#2953)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-teldat-snmp.md

| CRITICALSTATUS                   | Define the conditions to match for the status to be CRITICAL.  You can use the following variables: %{simStatus}, %{interfaceState}, %{cellId}, %{simIcc}, %{operator}, %{imsi} | '%{simStatus} =~ /LOCKED/ or %{simStatus} =~ /DETECTING/' |               |
| EXTRAOPTIONS                     | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                | --verbose                                                 |               |

</TabItem>
<TabItem value="Cpu" label="Cpu">

| Macro        | Description                                                                                                                                      | Valeur par défaut  | Obligatoire |
|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|

> | WARNING      |                                                                                                                                                  |                    |             |

Author: Je dois Maj

> | CRITICAL     |                                                                                                                                                  |                    |             |

Author: Je dois Maj

| --force-counters64       | Force to use 64 bits counters only. Can be used to improve performance.                                                                                                                                                                                                                    |
| --force-counters32       | Force to use 32 bits counters (even in snmp v2c and v3). Should be used when 64 bits counters are buggy.                                                                                                                                                                                   |
| --reload-cache-time      | Time in minutes before reloading cache file (default: 180).                                                                                                                                                                                                                                |

> | --oid-filter             | Choose OID used to filter interface (default: ifName) (values: ifDesc, ifAlias, ifName, IpAddr).                                                                                                                                                                                           |

Reviewer: ```suggestion
| --oid-filter             | Choose OID used to filter interfaces (default: ifName) (values: ifDesc, ifAlias, ifName, IpAddr).                                                                                                                                                                                           |
```

| --oid-display            | Choose OID used to display interface (default: ifName) (values: ifDesc, ifAlias, ifName, IpAddr).                                                                                                                                                                                          |
| --oid-extra-display      | Add an OID to display.                                                                                                                                                                                                                                                                     |

> | --display-transform-src  | Regexp src to transform display value.                                                                                                                                                                                                                                                     |

Reviewer: src?

> | --display-transform-dst  | Regexp dst to transform display value.                                                                                                                                                                                                                                                     |

Reviewer: dst?

| --statefile-concat-cwd | If used with the '--statefile-dir' option, the latter's value will be used as a sub-directory of the current working directory. Useful on Windows when the plugin is compiled, as the file system and permissions are different from Linux.   |
| --statefile-format     | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key        | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher     | Define the cipher algorithm to encrypt the cache (Default: 'AES').                                                                                                                                                                            |

> | --warning-uptime       | Threshold warning.                                                                                                                                                                                                                            |

Reviewer: ```suggestion
| --warning-uptime       | Warning threshold.                                                                                                                                                                                                                            |
```

> | --critical-uptime      | Threshold critical.                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| --critical-uptime      | Critical threshold.                                                                                                                                                                                                                           |
```

| --add-sysdesc          | Display system description.                                                                                                                                                                                                                   |

> | --force-oid            | Can choose your oid (numeric format only).                                                                                                                                                                                                    |

Reviewer: ?

> | --check-overload       | Uptime counter limit is 4294967296 and overflow. With that option, we manage the counter going back. But there is a few chance we can miss a reboot.                                                                                          |

Reviewer: ```suggestion
| --check-overload       | The uptime counter limit is 4294967296 centiseconds (~497 days). This option makes the plugin store the actual uptime by incrementing the number of revolutions of the counter when it resets to zero.                                                                                          |
```

> | --reboot-window        | To be used with check-overload option. Time in milliseconds (Default: 5000) You increase the chance of not missing a reboot if you decrease that value.                                                                                       |

Reviewer: ```suggestion
| --reboot-window        | To be used with check-overload option. Time in milliseconds (default: 5000). You increase the chance of not missing a reboot if you decrease that value.                                                                                       |
```

| --nagvis-perfdata        | Display traffic perfdata to be compatible with nagvis widget.                                                                                                                                                                                                                              |
| --interface              | Define which interfaces to monitor (number expected). Example: 1,2... (empty means 'check all interfaces').                                                                                                                                                                                                       |
| --name                   | Allows to use interface name with option --interface instead of interface OID index (can be a regexp).                                                                                                                                                                                       |

> | --speed                  | Set interface speed for incoming/outgoing traffic (in Mb).                                                                                                                                                                                                                                 |

Reviewer 2: ```suggestion
| --speed                  | Define the actual maximum traffic speed you can have on the interfaces in megabits per second.                                                                                                                                                                                                                                 |
```

> | --speed-in               | Set interface speed for incoming traffic (in Mb).                                                                                                                                                                                                                                          |

Reviewer 2: ```suggestion
| --speed-in               | Define the actual maximum incoming traffic speed you can have on the interfaces in megabits per second.                                                                                                                                                                                                                                          |
```

> | --speed-out              | Set interface speed for outgoing traffic (in Mb).                                                                                                                                                                                                                                          |

Reviewer 2: ```suggestion
| --speed-out              | Define the actual maximum outgoing traffic speed you can have on the interfaces in megabits per second.                                                                                                                                                                                                                                          |
```

| --statefile-cipher     | Define the cipher algorithm to encrypt the cache (Default: 'AES').                                                                                                                                                                            |
| --warning-uptime       | Warning threshold.                                                                                                                                                                                                                            |
| --critical-uptime      | Critical threshold.                                                                                                                                                                                                                           |
| --add-sysdesc          | Display system description.                                                                                                                                                                                                                   |

> | --force-oid            | Can choose your oid (numeric format only).                                                                                                                                                                                                    |

Reviewer 2: ```suggestion
| --force-oid            | Replace the default OID in order to retrieve the uptime (the OID must return a numeric value).  |
```
